.eslintrc

```diff
@@ -48,6 +49,8 @@
     ],
     "@typescript-eslint/no-unused-vars": "warn",
     "react/react-in-jsx-scope": "off",
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "off"
```

Comment:
I disabled this rule as it was throwing errors in ~70 different spots in the app - most of them were really opinionated about what was in a useEffect dependency array. Most of the suggestions either suggested we remove the dependency array entirely, or add items to the dependency array - it felt risky to do that in so many places. Open to feedback, though.

Reply:
Yeah, that rule has a lot of false positives.  Some of those 70 are likely actual bugs that are breaking the app in certain edge cases.  Maybe we can revisit this later.

Reply:
@jdorn I created an issue for this (https://github.com/growthbook/growthbook/issues/422) - If that's not the workflow we want, just let me know and I can delete it. I just don't want us to say we can revisit later and then forget about it.

Reply:
yep, issue is the right way to go